special-interest-groups/sig-weir/member-list.md

```diff
@@ -0,0 +1,20 @@
+# SIG Weir Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+
+## Tech Lead
+
+* [cx3ptr](https://github.com/cx3ptr), [伴鱼](https://www.ipalfish.com/)
+
+## Committers
```

Comment:
I think we can make eastfisher be contributor first, and when the contribution is enough, we can promote him to be committer?

Reply:
I have adjusted eastfisher from committer to reviewer, cause sig-weir is a totaly new sig, wo need two members to review each other.

Reply:
> I think we can make eastfisher be sig contributor first, and when the contribution is enough, we can promote him to be committer?


I have adjusted eastfisher from committer to reviewer, cause sig-weir is a totaly new sig, wo need two members to review each other.

Reply:
promote `@eastfisher` to be a reviewer sounds good to me